astroquery/vo_conesearch/core.py

```diff
@@ -0,0 +1,209 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from astropy.extern import six
+from astropy.extern.six import BytesIO
+from astropy.extern.six.moves import urllib
+
+from numbers import Number
+
+from astropy import units as u
+from astropy.coordinates import (BaseCoordinateFrame, ICRS, SkyCoord,
+                                 Longitude, Latitude)
+from astropy.io.votable import table
+
+from .exceptions import ConeSearchError, InvalidAccessURL
+from .vos_catalog import vo_tab_parse
+from ..query import BaseQuery
+from ..utils import commons
+
+# Import configurable items declared in __init__.py
+from . import conf
+
+__all__ = ['ConeSearch', 'ConeSearchClass']
+
+__doctest_skip__ = ['ConeSearchClass']
+
+
+class ConeSearchClass(BaseQuery):
+    """
+    The class for querying the Virtual Observatory (VO)
+    Cone Search web service.
+
+    Example
+    -------
+    >>> from astropy import units as u
+    >>> from astropy.coordinates import SkyCoord
+    >>> from astroquery.vo_conesearch import ConeSearch
+    >>> ConeSearch.query_region(SkyCoord.from_name('M31'), 5 * u.arcsecond)
+    <Table masked=True length=6>
+        objID           gscID2      ... compassGSC2id   Mag
+                                    ...                 mag
+        int64           object      ...     object    float32
+    -------------- ---------------- ... ------------- -------
+    23323175812944 00424433+4116085 ... 6453800072293      --
+    23323175812933 00424455+4116103 ... 6453800072282      --
+    23323175812939 00424464+4116092 ... 6453800072288      --
+    23323175812931 00424464+4116106 ... 6453800072280      --
+    23323175812948 00424403+4116069 ... 6453800072297      --
+    23323175812930 00424403+4116108 ... 6453800072279      --
+    """
+    TIMEOUT = conf.timeout
+    URL = conf.fallback_url
+    PEDANTIC = conf.pedantic
+
+    def __init__(self):
+        if not self.URL.endswith(('?', '&')):
+            raise InvalidAccessURL("url should already end with '?' or '&'")
```

Comment:
Do you mean removing that word from the error message, or just not check it at all?